docker/Dockerfile.custom_k8s_config

```diff
@@ -4,13 +4,14 @@
 # This Dockerfile expects a gzipped tarball called `agent_config.tar.gz` to be in the local
 # directory.  This tarball should contain the desired configuration files (`agent.json` and all
 # `.json` files in `agent.d`) relative to the `/etc/scalyr-agent-2` configuration directory.
-# You can produce such a tarball using a running Scalyr Agent container by issuing the following command:
+# You can extract such a tarball from a running Scalyr Agent container by issuing the following command:
```

Comment:
'extract a tarball' usually has a different meaning from what is meant here - the opposite in fact, because this command creates a new tarball from the config files, whereas extract normally means getting files out of the tarball, so I wonder if there's cause for confusion?

The config option is `--extract-config`, so it's extracting the *config*, and tarring it up, rather than extracting the tarball.

I wonder if 'produce' or 'create' might be the better wording here.  I'll leave it to you to decide.

Reply:
Good point. Will revert.